tl

```diff
@@ -498,6 +621,23 @@ local function type_check_and_load(filename, modules)
    return chunk
 end
 
+--------------------------------------------------------------------
+--                        RUN BUILD SCRIPT                        --
+--------------------------------------------------------------------
+local script = {}
+if tlconfig.build_file then
+   script = type_check_and_load(tlconfig.build_file,modules)()
```

Comment:
This should probably be
 - `pcall`ed to gracefully error out if there's an error in `build.tl`
 - manually type checked to, again, gracefully error out on user error

Basically we don't want user error to make a stacktrace point to inside of `tl`